src/View/Helper/ArrayHelper.php

```diff
@@ -25,10 +25,31 @@ class ArrayHelper extends Helper
      * Return array_combine of array using values as keys.
      *
      * @param array $arr The array.
+     *
```

Comment:
[minor] We usually don't add a blank line between `@param` and `@return`.

Reply:
something wrong with PHPCS I'm using ^_^